piano-hero/core/src/main/java/com/novoda/pianohero/Notes.java

```diff
@@ -0,0 +1,29 @@
+package com.novoda.pianohero;
+
+class Notes {
+
+    static final Note C4 = new Note(60);
+    static final Note C4_S = new Note(61);
```

Comment:
S for sharp - `C4_S` == `D4_b` but I didn't see the point in writing `C4_S_D4_b`